source/index.ts

```diff
@@ -360,6 +360,11 @@ is.nonEmptySet = <T = unknown>(value: unknown): value is Set<T> => is.set(value)
 is.emptyMap = (value: unknown): value is Map<never, never> => is.map(value) && value.size === 0;
 is.nonEmptyMap = <Key = unknown, Value = unknown>(value: unknown): value is Map<Key, Value> => is.map(value) && value.size > 0;
 
+/**
+`PropertyKey` is any value that can be used as an object key (string, number, or symbol)
+*/
```

Comment:
```suggestion
// `PropertyKey` is any value that can be used as an object key (string, number, or symbol)
```

Sorry for the nitpicking, but comments should either be:

1. oneliners:

```ts
// comment
someCode();
```

or 

2. multiliners:

```ts
/**
 * comment
 */
someCode()
```

Multi-line comments have a certain format to them (each intermediate line starting with `" * "`.

Since there's no real reason to have a multi-line comment, it's better we stick with a single line, IMO.

Reply:
My bad, I thought the initial request was for Typescript documentation, hence the `/** ... */` syntax.

If you need multiliner comment but not Typescript documentation, I would suggest the `/* ... */` syntax, with a single leading  `*`.

Reply:
Did you forget to upload the fix?